src/loadmill.ts

```diff
@@ -31,32 +34,32 @@ start()
     });
 
 async function start() {
-    
+
     let {
         wait,
         bail,
-        async,
         quiet,
         token,
         verbose,
         colors,
         report,
+        launchAllTestSuites,
         local,
         loadTest,
-        testSuite,
         additionalDescription,
         labels,
         args: [input, ...rawParams]
     } = program;
 
-    const logger = new Logger(verbose, colors);
+    const logger = getLogger({ verbose, colors });
 
     if (!token) {
         validationFailed("No API token provided.");
     }
 
     const parameters = toParams(rawParams);
 
+    const testSuite = !loadTest && !local;
```

Comment:
local is only for load?

Reply:
local is only for functional. If the user didnt want local and didnt want load so it's suite